examples/pigweed-app/nrfconnect/CMakeLists.txt

```diff
@@ -18,6 +18,10 @@ cmake_minimum_required(VERSION 3.13.1)
 get_filename_component(CHIP_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/third_party/connectedhomeip REALPATH)
 get_filename_component(NRFCONNECT_COMMON ${CHIP_ROOT}/examples/platform/nrfconnect REALPATH)
 
+if(${BOARD} STREQUAL "nrf52840dongle_nrf52840")
+list(INSERT OVERLAY_CONFIG 0 ${CHIP_ROOT}/config/nrfconnect/overlay-usb_support.conf)
```

Comment:
Note that if the file wasn't generic you could just put it in <app-dir>boards/nrf52840dongle_nrf52840.conf and it would be applied automatically.